solr/solr-ref-guide/src/_config.yml.template

```diff
@@ -91,6 +91,11 @@ asciidoctor:
     <<: *solr-attributes-ref
     attribute-missing: "warn"
     icons: "font"
-    source-highlighter: "rouge"
-    rouge-theme: "thankful-eyes"
     stem:
+    source-highlighter: "rouge"
+    # Options from: https://github.com/jirutka/asciidoctor-rouge
+    rouge-css: "class"
+    # class option requires css file generated by build/.gems/gems/rouge-vx.y.z/bin/rougify style style_name
+    # other option: style for inline styles, which is larger in size
+    # rouge-css: "style"
+    # rouge-style: "github"
```

Comment:
I'm not sure I understand the point of the commented sections here?

Reply:
The styles generated for the source blocks (highlights) can be inlined or in a separate css, which needs to be generated. Due to the support for - I guess - dark mode - the inline styles make the documents somewhat larger. So, CSS looked like a better option.

Also, the previous style was spelled incorrectly and therefore falling-back to default style. I've just made of this explicit after spending time trying to figure out the actual situation.